cgcnn2/util.py

```
@@ -463,3 +461,29 @@ def load_state_dict(self, state_dict: dict[str, torch.Tensor]) -> None:
+    mae   = checkpoint.get("best_mae_error")
+
+    metrics = []
+    if mse is not None:
+        metrics.append(f"MSE={mse:.4f}")
+    if mae is not None:
+        metrics.append(f"MAE={mae:.4f}")
+
+    metrics_str = ", ".join(metrics) if metrics else "N/A"
+
+    print(
+        f"=> Loaded model from '{args.model_path}' "
```

Ruff (0.11.9)

Undefined name `args`

(F821)